src/NuGet.Clients/NuGet.CommandLine/PackageSourceBuilder.cs

```diff
@@ -1,13 +1,15 @@
-﻿using System.Collections.Generic;
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
 using NuGet.Configuration;
 
 namespace NuGet.CommandLine
 {
     internal static class PackageSourceBuilder
     {
-        internal static Configuration.PackageSourceProvider CreateSourceProvider(Configuration.ISettings settings)
+        internal static PackageSourceProvider CreateSourceProvider(ISettings settings)
         {
-            return new Configuration.PackageSourceProvider(settings);
+            return new PackageSourceProvider(settings, enablePackageSourcesChangedEvent: false);
```

Comment:
Is this because all the consumers are CLI commands?

Reply:
ultimately it doesn't matter becuase the CLI commands are short lived processes, so leaks are limited in impact. I largely made the change so I could use IntelliCode to find all references to the constructor without the bool parameter, to ensure I find all usage that does matter (which happened to only be 1).